plugins/inputs/knx_listener/README.md

```diff
@@ -17,6 +17,11 @@ underlying "knx-go" project site (<https://github.com/vapourismo/knx-go>).
   ## Address of the KNX-IP interface.
   service_address = "localhost:3671"
 
+  ## Used transport protocol.
+  ## Can be either "UDP" (default) or "TCP".
+  ## TCP is currently only in tunnel mode supported.
```

Comment:
```suggestion
  ## TCP is only supported in tunnel mode.
```
There is no TCP routing (multicast), so this is not going to change 😉

Reply:
OMG... Shame on me :-)